src/app/model/movies-dto.ts

```diff
@@ -0,0 +1,8 @@
+import {Movie} from "./movie/movie";
+
+export interface MoviesDto {
+  totalAmount : number;
```

Comment:
totalAmount can be inferred from movies.length

Reply:
I get this object from backend service. These fields are defined there